src/lib/structures/DbSet.ts

```diff
@@ -0,0 +1,130 @@
+import { connect } from '@orm/dbConfig';
+import { BannerEntity } from '@orm/entities/BannerEntity';
+import { CommandCounterEntity } from '@orm/entities/CommandCounterEntity';
+import { DashboardUserEntity } from '@orm/entities/DashboardUserEntity';
+import { GiveawayEntity } from '@orm/entities/GiveawayEntity';
+import { GuildEntity } from '@orm/entities/GuildEntity';
+import { ModerationEntity } from '@orm/entities/ModerationEntity';
+import { RpgBattleEntity } from '@orm/entities/RpgBattleEntity';
+import { RpgClassEntity } from '@orm/entities/RpgClassEntity';
+import { RpgGuildEntity } from '@orm/entities/RpgGuildEntity';
+import { RpgGuildRankEntity } from '@orm/entities/RpgGuildRankEntity';
+import { RpgItemEntity } from '@orm/entities/RpgItemEntity';
+import { RpgUserEntity } from '@orm/entities/RpgUserEntity';
+import { RpgUserItemEntity } from '@orm/entities/RpgUserItemEntity';
+import { StarboardEntity } from '@orm/entities/StarboardEntity';
+import { SuggestionEntity } from '@orm/entities/SuggestionEntity';
+import { TwitchStreamSubscriptionEntity } from '@orm/entities/TwitchStreamSubscriptionEntity';
+import { UserProfileEntity } from '@orm/entities/UserProfileEntity';
+import { UserCooldownEntity } from '@orm/entities/UserCooldownEntity';
+import { UserRepository } from '@orm/repositories/UserRepository';
+import { ClientRepository } from '@orm/repositories/ClientRepository';
+import { MemberRepository } from '@orm/repositories/MemberRepository';
+import { BrandingColors } from '@utils/constants';
+import type { Connection } from 'typeorm';
+import type { Message } from 'discord.js';
+
+
```

Comment:
```suggestion

import type { Connection } from 'typeorm';
import type { Message } from 'discord.js';

```

Reply:
We don't really space out type imports because they get remove by VSCode's organize imports anyway.